docs/docs/.vuepress/configs/envVars.js

```diff
@@ -6,8 +6,168 @@ module.exports = {
   MEMORY_LIMIT: "256MB",
   MEMORY_ABSOLUTE_LIMIT: "10GB",
   EMAIL_PAYLOAD_SIZE_LIMIT: "30MB",
-  FREE_INSPECTOR_EVENT_LIMIT: "100",
-  PAID_INSPECTOR_EVENT_LIMIT: "1,000",
+
+  limits: {
```

Comment:
I think we're likely to make changes to tiers over the next few months as we learn more, and I'm concerned the `limits` data here may be quickly out of date. Since you're not using most of it anymore, should we remove?

Reply:
🧹